setup.py

```diff
@@ -58,6 +58,9 @@
             "google-cloud-bigquery[pandas]",
             "google-cloud-bigquery-storage",
         ],
+        "bigframes": [
+            "bigframes @ git+https://github.com/googleapis/python-bigquery-dataframes.git@c71ec093314409cd4c7a52a713dbd6164fbbd792"
```

Comment:
why is this pinned to git and not regular big frames?

Reply:
Waiting on a version release upstream. 

https://github.com/googleapis/python-bigquery-dataframes/pull/1278 (merged)
https://github.com/googleapis/python-bigquery-dataframes/pull/1283 (followup on googles side)

Reply:
maybe let's wait til a release?